app/services/market_analysis_core.py

```
@@ -259,27 +259,42 @@ async def technical_analysis(
+                "success": overall_success,
+                "function": "technical_analysis",
                 "data": analysis_results,
-                "metadata": {
-                    "symbols_analyzed": len(symbol_list),
-                    "timeframe": timeframe,
-                    "indicators_used": indicator_list,
-                    "response_time_ms": round(response_time * 1000, 2),
-                    "timestamp": datetime.utcnow().isoformat()
-                }
+                "metadata": metadata,
             }
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)